sygnal.yaml.sample

```diff
@@ -231,12 +225,23 @@ apps:
   #  # Defaults to 512 if unset.
   #  #
   #  #inflight_request_limit: 512
+  #
+  #  # Specifies whether to use the production or sandbox APNs server. Note that
+  #  # sandbox tokens should only be used with the sandbox server and vice versa. 
+  #  #
+  #  # Valid options are:
+  #  #   * production
+  #  #   * sandbox
+  #  #
+  #  # The default is 'production'.
+  #  #platform: sandbox
```

Comment:
Maybe we want to start making this like the synapse one where we say "Uncomment the following to use sandbox mode" or whatever?

Reply:
I am definitely not opposed to that!